exercises/01.fundamentals/B04/D03/ex-bonus.js

```diff
@@ -0,0 +1,66 @@
+/* 
+total-lines -> n%2==1;
+n1 -> ---*---
+n2 -> --------
+n3 -> --*-*--
+n4 -> --------
+n5 -> -*---*-
+n6 -> --------
+n7 -> *******
+*/
+
+function rCh(char, repeat) {
+    let output = ""
+    for (let x = 0; x < repeat; x++) {
+        output += `${char}`;
+    }
+    return output;
+}
+
+// Item 01
+console.log(`\n== Item01 ==`)
+function funnyTriangle(n, char = '*', space = ' ') {
+    if (n % 2 == 1 && n > 0) {
+        for (let x = 1; x <= n; x += 2) {
+            if (x === 1) {
+                console.log(rCh(space, (n - 1) / 2) + char + rCh(space, (n - 1) / 2));
+                continue;
+            }
+            if (x == n) { console.log(rCh(char, n)); continue; };
+
+            let betweenAsteriscs = x - 2;
+            let startSpaces = (n - (betweenAsteriscs + 2)) / 2;
+            console.log(
```

Comment:
Murilo, creio que o `${char} é desnecessário nesse contexto, você pode usar apenas char, caso queira usar o string literals recomendo usar em todo o resultado. Faz sentido para você?
```suggestion
            console.log(`${rCh(space, startSpaces)}${char}${rCh(space, betweenAsteriscs)}${char}`
```

Reply:
De fato, fica uma mais conciso usar o _template literals_. Muito obrigado pela sugestão.